### PR TITLE
Implemented action in index method of StudentController and added Vagrant file into the root folder

### PR DESCRIPTION
**Pull request by vladimir-vuckovic-devtech**
Purpose of this MR is displaying all students from database

### CODE REVIEW

**Comment by dejan-babic** [comment]
Please use the following template no matter how simple the MR is. Trust me this might seem irritating but it needs to get in to the blood flow.

What's the purpose of this MR?
(Insert The Description of the Purpose of this Branch here)
Impact Analysis
(What will this possibly affect?)
Where should the tester start?
(Hints tips or tricks regarding how to test this, things to watch out for, etc)
How should this be manually tested?
(Steps to test?)
Any background context you want to provide?
(?)
What are the relevant tickets?
(Is this related to a ticket?)
Screenshots (if appropriate)
(N/A)

**Comment by dejan-babic** [comment]
@vladimir-vuckovic-devtech 
the `storage` folder should not be part of the repo, so first you need to remove it's content, and put it in .gitignore so it is not tracked anymore

**Comment by dejan-babic** [comment]
@vladimir-vuckovic-devtech 
Also, i don't see the `.env` file, and if found the debug should be set to true.
Perhaps you have it locally only ?

**Comment by vladimir-vuckovic-devtech** [comment]
What's the purpose of this MR?
Purpose of this MR is displaying all students from database.

## Impact Analysis

Where should the tester start?
(Hints tips or tricks regarding how to test this, things to watch out for, etc)
go to /student page and there should be shown table with all student names and passwords.

(Steps to test?)

## Any background context you want to provide?

What are the relevant tickets?
(Is this related to a ticket?)
"Create index action - simple bootstrap table for now"

Screenshots (if appropriate)
(N/A)

**Comment by dejan-babic** [comment]
You need to improve error handling, cover next cases:
- The database is not reachable
- The credentials are wrong
- The Table does not exist 
- The Table is empty

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -44,7 +44,11 @@ public function report(Exception $exception)
      */
     public function render($request, Exception $exception)
     {
-        return parent::render($request, $exception);
+        if($exception instanceof \PDOException){
+            return response()->view('no_database',['error_message' => "Konekcija ka bazi nije uspesno uspostavljena"]);
+        }
```
Comment (dejan-babic) [inline app/Exceptions/Handler.php:49]: I hate it also, but it has to be english

File: app/Http/Controllers/StudentController.php [hunk 2]
```diff
@@ -15,7 +19,17 @@ class StudentController extends Controller
...
     public function index()
     {
-        echo "from index";
+        if(!Schema::hasTable("students")) {
+            throw new \Exception("Trazena tabela ne postoji u bazi.");
+        }
+
+        $students = Student::all();
+        if ($students->isEmpty()) {
+            throw new \Exception("Tabela je prazna.");
```
Comment (dejan-babic) [inline app/Http/Controllers/StudentController.php]: No need to throw if empty, but rather change the blade template and draw the else case (I remember that you check are the students set)

**Pull request merged**